x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -75,7 +75,7 @@ export function ProcessTreeNode({
   const { tty } = processDetails.process;
 
   const renderChildren = () => {
-    const children = process.getChildren(showGroupLeadersOnly);
+    const children = process.getChildren(!showGroupLeadersOnly);
```

Comment:
this call will be updated to take a verboseMode:boolean property. @animehart is working on this change as part of the verbose mode and timestamp toggle work.